packages/docusaurus-preset-bootstrap/package.json

```diff
@@ -8,7 +8,8 @@
     "access": "public"
   },
   "dependencies": {
-    "@docusaurus/plugin-content-pages": "^2.0.0-alpha.50"
```

Comment:
Why remove this? It's needed for the index page at http://localhost:3000

Reply:
Yeah, when I removed its because it was trying to understand how is the flow of the blog, and I decided to see leave only the blog, but don't make sense not include again.